### ISSUE TITLE
Can we align the shortnames with MS abbreviations

### ISSUE THREAD

**Issue by jag-sandhu** [microsoft/CloudAdoptionFramework#54]
Can the shortnames be aligned to the MS recommended shortnames/abbreviations where possible:

https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

I understand this can be done manually by editing the resources.csv, but I would be nice to have everything aligned and consistent where possible.

**Comment by jamasten** [comment]
@jag-sandhu I assume your issue is with the Azure Naming Tool since you are referencing the resources.csv file.  We recently updated the resource short names to align with Enterprise Scale Landing.  The short names on the link you shared will be updated to align with the Azure Naming Tool here soon.  We are waiting on the CAF team to make our changes public.

**Comment by jamasten** [comment]
Here's the PR for your reference: https://github.com/MicrosoftDocs/cloud-adoption-framework/pull/615

**Comment by jag-sandhu** [comment]
@jamasten Thats great!
Thanks for clarifying and pointing me to the PR for visibility.
